src/styles/main-styles.scss

```diff
@@ -1,15 +1,39 @@
-$color-grey: #a3a3a3;
-$color-layout: rgba(171, 171, 171, 0.55);
-$color-dark-brown: #601919;
-
-body {
-    width: 99%;
-}
-
 .additional-margin-top {
     margin-top: 90px;
 
     &--less {
         margin-top: 45px;
     }
 }
+
+.todo-element {
+    min-height: 60px;
```

Comment:
Avoid using `px` in styles, much better would be using of `em`, thanks that all elements on site would be scaled according to font size